src/main/java/seedu/address/model/person/Patient.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a patient that requires home-visit in Healthcare Xpress book.
+ */
+public class Patient extends Person {
+
+    private static final String MESSAGE_FOR_EMPTY_DATETIME = "Home Visit date and time has not been set yet.";
+    private final List<DateTime> dateTimes = new ArrayList<>();
+
```

Comment:
Should the Patient class include a boolean to keep track of whether the patient has been visited? The class can also have markAsVisited() and markAsUnvisited() methods to toggle this.